Web_pet1/Web_pet1/WeatherForecast.cs

```diff
@@ -0,0 +1,13 @@
+namespace Web_pet1
+{
+    public class WeatherForecast
+    {
+        public DateTime Date { get; set; }
+
+        public int TemperatureC { get; set; }
+
+        public int TemperatureF => 32 + (int)(TemperatureC / 0.5556);
+
+        public string? Summary { get; set; }
+    }
+}
```

Comment:
Please add an empty line to avoid this warning: 
![image](https://user-images.githubusercontent.com/17098967/185183249-d32e882a-8fe6-41ce-88f0-906b8a5a1ab5.png)